src/lib/config.js

```diff
@@ -36,4 +36,5 @@ module.exports = {
   SERVICE_URL: process.env.SERVICE_URL,
   TEMPLATE_CACHING: process.env.TEMPLATE_CACHING,
   SERVICE_DOMAIN: process.env.SERVICE_DOMAIN || "localhost",
+  LANGUAGE_TOGGLE_ENABLED: process.env.LANGUAGE_TOGGLE === true || false,
```

Comment:
I believe the values of the env vars are actually strings (as aws interprets them) so this should be `LANGUAGE_TOGGLE_ENABLED: process.env.LANGUAGE_TOGGLE === "true"` ?